src/theme.rs

```diff
@@ -495,7 +495,7 @@ pub trait Theme {
     }
 
     /// Returns the multiline note message rendering.
-    fn format_note(&self, prompt: &str, message: &str) -> String {
+    fn format_note(&self, is_outro: bool, prompt: &str, message: &str) -> String {
```

Comment:
It's gonna break backward compatibility for Themes. You can do the following:

```rust
1. format_note_generic(is_outro, prompt, message)
2. format_note(prompt, message) -> calls format_note_generic(false, prompt, message)
3. format_outro_note(true, prompt message) -> calls format_note_generic(true, prompt, message)
```

Reply:
Fixed in latest push